auth-server/src/utils/getAddressesFromUserId.ts

```diff
@@ -0,0 +1,13 @@
+import Address from '../model/Address';
+
+/**
+ * Get Addresses from userId
+ */
+export default async (userId: number): Promise<Address[]> => {
+	const addresses = await Address
+		.query()
+		.where('user_id', userId);
```

Comment:
you need to filter for the one that are "linked". BTW "linked" is probably a not so great name. "verified" would be better.

Reply:
done